data/api/apiAccountResponse.go

```diff
@@ -2,14 +2,15 @@ package api
 
 // AccountResponse is the data transfer object to be returned on API when requesting an address data
 type AccountResponse struct {
-	Address         string `json:"address"`
-	Nonce           uint64 `json:"nonce"`
-	Balance         string `json:"balance"`
-	Username        string `json:"username"`
-	Code            string `json:"code"`
-	CodeHash        []byte `json:"codeHash"`
-	RootHash        []byte `json:"rootHash"`
-	CodeMetadata    []byte `json:"codeMetadata"`
-	DeveloperReward string `json:"developerReward"`
-	OwnerAddress    string `json:"ownerAddress"`
+	Address         string            `json:"address"`
+	Nonce           uint64            `json:"nonce"`
+	Balance         string            `json:"balance"`
+	Username        string            `json:"username"`
+	Code            string            `json:"code"`
+	CodeHash        []byte            `json:"codeHash"`
+	RootHash        []byte            `json:"rootHash"`
+	CodeMetadata    []byte            `json:"codeMetadata"`
+	DeveloperReward string            `json:"developerReward"`
+	OwnerAddress    string            `json:"ownerAddress"`
+	Pairs           map[string]string `json:"pairs,omitempty"`
```

Comment:
Is it ok that this is a map and not a slice?
Would it ever be of interest to have them in a specific order?

Reply:
Also, if the query param is `withKeys`, why is this field called `Pairs`?